### PR TITLE
introduce s82

### PR DESCRIPTION
**Pull request by StevenClontz**

### CODE REVIEW

**Comment by StevenClontz** [comment]
This completes a search query made at https://math.stackexchange.com/questions/3155216/example-limit-point-compact-negcountably-compact-neglindel%c3%b6f/4744910#4744910

**Comment by prabau** [comment]
It should not matter for the results.  But do we want to make the number of summands definite, like continuum many, at least in the description, or leave it as is?

File: spaces/S000082/properties/P000062.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+space: S000082
+property: P000062
+value: false
+refs:
+- mathse: 3155216
+  name: "Example: limit point compact + ¬countably compact+ ¬Lindelöf"
+---
+
+The clopen cover using each of the uncountable copies of {S42} has no countable subcover.
```
Comment (prabau) [inline spaces/S000082/properties/P000062.md:10]: As written, this is the justification that the space is not Lindelof.  For weakly Lindelof, need to modify a little.

**Comment by prabau** [comment]
Not relevant here, but a question about S42 that this is based on.
https://topology.pi-base.org/spaces/S000042/properties/P000068 Doesn't the mathse reference actually show that the space is Markov Rothberger?

**Comment by prabau** [comment]
The mathfrak in the name of the space seems a little clunky when trying to do a search in the pi-base search box.  Would it be possible to change the space name to "Sum of continuum-many Right-ordered reals" (even shorter than before without "topological")?

All good otherwise.

**Comment by prabau** [comment]
Sorry, one last nitpick.  I see that you changed the capitalization in the middle, from "Sum of continuum-many Right-ordered reals" to "Sum of continuum-many right-ordered reals".  I thought capitalizing "Right" in the middle was kind of nice as an indication that we are referring to another space somewhere.  But I am reading too much into this, I can go with the latest.  What do you prefer?

**Comment by StevenClontz** [comment]
works for me

**Review by prabau — APPROVED** [review]

**Pull request merged**